components/Articles.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { useChannel } from "@ably-labs/react-hooks";
+import { formatDate } from "../lib/dates";
+import ArticlePreview from "./ArticlePreview";
+import styles from "../styles/Home.module.css";
+
+export default function Articles(props) {
+  let inputBox = null;
+  const defaultMessages = props.history || [];
+
+  const [headlineText, setHeadlineText] = useState("");
+  const [headlines, updateHeadlines] = useState(defaultMessages);
+  const [_, ably] = useChannel("[?rewind=3]headlines", (headline) => {
```

Comment:
Using rewind instead of the history API, fewer roundtrips to the server, also ensures correct message order.